### ISSUE TITLE
Datepicker improve a11y

### ISSUE THREAD

**Issue by diwic** [oruga-ui/oruga#357]
### Overview of the problem

**Oruga** version: 0.5.5
**Vuejs** version: 2.6
**OS/Browser**: Ubuntu/Firefox 99.0

### Description

Hi! I'm working on improving accessibility and run into something I'm not sure how to fix myself. I'm using Firefox's checker and it errors on "[Clickable elements must be focusable and should have interactive semantics](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard#clickable_elements_must_be_focusable_and_should_have_interactive_semantics)", the component being `o-ctrl-input`.

### Steps to reproduce

 1. Open https://oruga.io/components/datepicker.html in Firefox.
 2. Open Firefox devtools and choose the accessibility tab. 
 3. Choose to check for all issues

### Expected behavior

No errors/warnings

### Actual behavior

Many errors/warnings. But the one I don't know how to fix in particular, is the one on `o-ctrl-input` that says `Clickable elements must be focusable and should have interactive semantics`.

**Comment by jtommy** [comment]
@diwic I never saw that error, to be honest. Anyway any improvements about accessibility are welcome

**Comment by astagi** [comment]
hey @diwic  any update on this task?

**Comment by diwic** [comment]
![bild](https://user-images.githubusercontent.com/9721510/214009354-1acc9a44-4ce6-4cbc-9d6d-ebe0a818a76e.png)

The error is still there; but it's now on the `div.o-drop__trigger` element.
